lib/internal/util.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const prefix = '(node) ';
+const prefix = `(node:${process.pid}) `;
```

Comment:
@Fishrock123 I speculate that `process` itself is not defined in this case,
```
const prefix = `(node:${process.pid}) `;
                        ^
ReferenceError: process is not defined
```
so I can't use `process.title` or `process.release.name`